bin/make_cfgs.py

```diff
@@ -40,7 +40,7 @@
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
 
-def reorder_ip(ip):
+def reverse_ip(ip):
```

Comment:
Because the ip is supposed to be always a string represented IPv4, consider to use this single line instead of regex:
```
   return '.'.join(reversed(ip.split('.')))
```

Reply:
done